docs/Documentation/Development/Programming_Models/mpi.md

```diff
@@ -1,6 +1,57 @@
-* Cray MPICH
-* Open MPI
-* Intel MPI
-* MPI in NVHPC 
+# OpenMpi
+**Documentation:** [ link to documentation](https://www.open-mpi.org)
```

Comment:
I think for the link text we should name the documentation or put just the link, rather than put "link to documentation". Same for the other sections. 

Something like one of these examples:
**Documentation:** [ https://www.open-mpi.org](https://www.open-mpi.org)
**Documentation:** [ Open MPI](https://www.open-mpi.org)